components/layout/navbar/NavbarRoot.tsx

```diff
@@ -1,5 +1,6 @@
 'use client';
 
+import Link from 'next/link';
```

Comment:
```suggestion
import { Link } from '@/navigation';
```

i18n 때문에 Link, redirect, usePathname, useRouter 필요하면 위 파일에서 가져다쓰면 됨!